Homework 1

larna opened the pull request.

larna committed:
Вопросы с вариантами ответов читает из ресурса. Создает тест для пользователя. Показывает пользователю вопросы и ожидает ответов на них. После прохождения теста показывает результат прохождения - какие ответы правильные, какие неправильные.

larna committed:
Добавила тесты, немного изменила структуру.
Приложение не реагирует на пустые строки в файле ресурсов.
Работает для Ivanov.
Задает вопросы 2х типов - с возможностью выбора ответа из списка и вопросы, где ответ требуется вводить. Обрабатывает ситуацию когда ввели ерунду. В конце показывает результат теста - правильные/неправильные ответы пользователя.
Упаковывается в jar.

kataus commented on homework-1-students-testing/src/main/java/ru/otus/spring/domain/testing/Question.java:
```diff
@@ -0,0 +1,104 @@
+    /**
+     * Собственно сам вопрос в виде текста
+     */
+    @Getter
```
А почему только в одном месте ломбок? В классе много чего можно убрать с помощью ломбок

larna replied:
Я до этого lombok не использовала, здесь его попробовала. 
Если взяла в проект, то применить его везде, где возможно?

larna committed:
Добавила аннотации lombok в пакете domain.
Добавила интерфейс TestController и класс TestControllerConsole, который занимается обслуживанием консольного интерфейса.
В Main теперь поднимается контекст, из контекста получаем bean TestController и запускаем тестирование студентов.
